### DISCUSSION TITLE
Query on PDF Generation Service

### DISCUSSION THREAD

**Discussion by atulgupta2024** [egovernments/Digit-Core#409] [DIGIT Platform]
1. If I have a table that is rendered in the PDF, is it possible to create table headings at every page, if the table overflows to next page?
2. Pdf-service repo does not have context path defined so causing issues in deployment
3. Is page numbering possible ?
4. Can we add footers and headers in each page ?

Documentation at Digit Urban has more and better explanation than Digit Core. It may be a good idea to keep the documentation in sync or point to Digit Urban as a place to read more.

**Comment by subhashini-egov** [comment]
The PDF service is based on PDF make which takes in a document template (Mustache). So everything pdfMake supports the pdfService should support. Here is the documentation for PDFMake - http://pdfmake.org/#/   It explains how to add header, footer and table.
